### ISSUE TITLE
Safari & minicart

### ISSUE THREAD

**Issue by slackerzz** [magento/magento2#8178]
### Preconditions
1. 2.0.x & 2.1.2 (both CE & EE)
2. blank theme
3. Safari (both mac and ios)

### Steps to reproduce
1. click the minicart icon to open the dropdown

### Expected result

1. desktop
![schermata del 2017-01-18 14-38-15](https://cloud.githubusercontent.com/assets/3061752/22066138/d8fbaf20-dd8b-11e6-9ef3-7d264c4c3a6a.png)
2. mobile
![schermata del 2017-01-18 14-38-40](https://cloud.githubusercontent.com/assets/3061752/22066142/e02c810c-dd8b-11e6-842d-e47dd407e27f.png)

### Actual result

1. desktop
![bs_macsie_safari_broken](https://cloud.githubusercontent.com/assets/3061752/22065960/e50b8322-dd8a-11e6-8e15-f45011095cb6.jpg)
2. mobile
![bs_ios_mobile_iphone 7 plus-10 0](https://cloud.githubusercontent.com/assets/3061752/22066060/69569f40-dd8b-11e6-9027-6f0e32af2086.jpg)

I've found a really strange behavior, when the minicart dropdown is shown any style modification to the page through the inspector will fix the misplaced elements.

**Comment by rudik99** [comment]
Also the same for 2.1.3

**Comment by xmav** [comment]
@slackerzz Thanks for reporting this issue. 
This issue is already addressed in internal ticket MAGETWO-63089. I’ve added your report on this issue to that ticket.

**Comment by slackerzz** [comment]
@xmav since it's not possible for us to track your internal tickets, can you tell if there is a solution/workaround?

**Comment by xmav** [comment]
@slackerzz this issue will be closed in sync with internal ticket

**Comment by Quinten** [comment]
You can try to tweak the `position` css of `.minicart-wrapper .ui-dialog`. Try `static` or `absolute`

I have something like this somewhere in some theme css:

```
.minicart-wrapper .ui-dialog {
    position: absolute;
    width: 0px;
    height: 0px;
    top: 100%;
    left: 100%;
}
```

Still gives a strange glitching flash in IE9 and sometimes in safari on mac, but then it's fine. Doesn't do this on ios though.

**Comment by git-reset--hard** [comment]
Setting static width for the mini cart icon wrapper fixed this issue for me:
    .minicart-wrapper {
        width: 22px;
    }

**Comment by patruvlad** [comment]
My fix is something close to what @Quinten did except no positioning since the child has the style we need ( if you correctly use .lib-dropdown()
```
.minicart-wrapper .ui-dialog {
    width: 0;
}
```

Tested crossbrowser and works without flickering

**Comment by slackerzz** [comment]
It would be nice to know what happened to MAGETWO-63089

**Comment by Flamestyle** [comment]
@Patruvlad, Could you write where exactly these lines were written?

**Comment by patruvlad** [comment]
@Flamestyle sure thing, you need to write the following codes of line in `Magento_Checkout/web/css/source/module/_extend.less` if you have a custom theme with inheritance from the Magento_Blank. If you have a custom file something like `_minicart_extend.less` that is included in the `_extend.less` set it in the custom style file

```
.minicart-wrapper{  
        > .mage-dropdown-dialog {
            width: 0;
        }
}
```

**Comment by magento-engcom-team** [comment]
@slackerzz, thank you for your report.
We've created internal ticket(s) MAGETWO-63985 to track progress on the issue.

**Comment by undefined-olha** [comment]
Hi, everyone! 
You can use this [commit](https://github.com/magento/magento2/commit/41ceeb6208c461ac8e1d261d3cd3f5b6b59e19c3#diff-840626449b4aa53dc3745a0e53e158b9) as patch to fix an issue)

**Comment by okorshenko** [comment]
The issue has been fixed in 2.1-develop branch and will be available with next patch release
